site/public/css/simple-grading.css

```diff
@@ -20,4 +20,27 @@
 
 .sticky-top label, .sticky-top .not-sticky {
     display: none !important;
-}
\ No newline at end of file
+}
+
+.simple-full-credit {
+    color: #149bdf;
```

Comment:
Are these colors standardized? (if not, should they be?)

Reply:
Yeah I noticed the same thing. I took a look and saw the javascript was using these color tags rather than a defined color variable. Now that this is in two places it could be something good to standardize.

Reply:
Oh yea, I meant to do that and forgot. I'll make those changes.